platform/android/MapLibreAndroidTestApp/src/main/java/org/maplibre/android/testapp/activity/benchmark/BenchmarkActivity.kt

```diff
@@ -195,20 +199,18 @@ class BenchmarkActivity : AppCompatActivity() {
 
     private fun setupMapView() {
         mapView = findViewById<View>(R.id.mapView) as MapView
-        if (measureFrameTime) {
-            mapView.addOnDidFinishRenderingFrameListener { _: Boolean, frameEncodingTime: Double, frameRenderingTime: Double ->
-                encodingTimeStore.add(frameEncodingTime * 1e3)
-                renderingTimeStore.add(frameRenderingTime * 1e3)
-            }
+        mapView.addOnDidFinishRenderingFrameListener { _: Boolean, frameEncodingTime: Double, frameRenderingTime: Double ->
+            encodingTimeStore.add(frameEncodingTime * 1e3)
+            renderingTimeStore.add(frameRenderingTime * 1e3)
+            numFrames++;
```

Comment:
Looks like we could report both in one report?

Reply:
No, because of `maplibreMap.setSwapBehaviorFlush(measureFrameTime)`. For sync rendering we export all 3: fps, encoding and rendering, also for async rendering. It will gather more data for a better understanding.

Reply:
I've changed the variable `measureRenderingTime` to `syncRendering`.